src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -88,7 +88,7 @@ public enum PSSessionType
         DefaultRemoteShell = 0,
 
         /// <summary>
-        /// Default Workflow endpoint.
```

Comment:
Register-PSSessionConfiguration cmdlet has PSSessionType parameter and I want get confirmation that we can make the breaking change:
- remove the value
- or remove the type at all

I'd remove the PSSessionType type because we removed Workflow support and parser error will be ok.